test/python/circuit/test_rust_equivalence.py

```diff
@@ -74,63 +74,12 @@ def test_definitions(self):
-                                [rs_def.find_bit(x).index for x in rs_inst.qubits],
-                            )
-                        # In the following cases, Rust uses P but python still uses U1 and U3
-                        elif (
-                            name in {"z", "s", "sdg", "t", "tdg", "rz", "u1", "crx"}
-                            and rs_inst.operation.name == "p"
-                        ):
-                            if py_inst.operation.name == "u1":
-                                self.assertEqual(py_inst.operation.name, "u1")
-                                self.assertEqual(rs_inst.operation.params, py_inst.operation.params)
-                                self.assertEqual(
-                                    [py_def.find_bit(x).index for x in py_inst.qubits],
-                                    [rs_def.find_bit(x).index for x in rs_inst.qubits],
-                                )
-                            else:
-                                self.assertEqual(py_inst.operation.name, "u3")
-                                self.assertEqual(
-                                    rs_inst.operation.params[0], py_inst.operation.params[2]
-                                )
-                                self.assertEqual(
-                                    [py_def.find_bit(x).index for x in py_inst.qubits],
-                                    [rs_def.find_bit(x).index for x in rs_inst.qubits],
-                                )
-                        # In the following cases, Rust uses CP but python still uses CU1
-                        elif name in {"csx"} and rs_inst.operation.name == "cp":
-                            self.assertEqual(py_inst.operation.name, "cu1")
-                            self.assertEqual(rs_inst.operation.params, py_inst.operation.params)
-                            self.assertEqual(
-                                [py_def.find_bit(x).index for x in py_inst.qubits],
-                                [rs_def.find_bit(x).index for x in rs_inst.qubits],
-                            )
-                        else:
-                            self.assertEqual(py_inst.operation.name, rs_inst.operation.name)
-                            self.assertEqual(rs_inst.operation.params, py_inst.operation.params)
-                            self.assertEqual(
-                                [py_def.find_bit(x).index for x in py_inst.qubits],
-                                [rs_def.find_bit(x).index for x in rs_inst.qubits],
-                            )
+                        self.assertEqual(py_inst.operation.name, rs_inst.operation.name)
+                        self.assertEqual(rs_inst.operation.params, py_inst.operation.params)
+                        self.assertEqual(
+                            [py_def.find_bit(x).index for x in py_inst.qubits],
+                            [rs_def.find_bit(x).index for x in rs_inst.qubits],
+                        )
```

Comment:
Now that the Python-side definitions and Rust-side definitions are exactly identical, we do not need the special cases above.